tests/client/amo/components/TestSearchResult.js

```diff
@@ -1,22 +1,63 @@
 import React from 'react';
 import {
+  findRenderedComponentWithType,
   findRenderedDOMComponentWithClass,
   renderIntoDocument,
 } from 'react-addons-test-utils';
 
+import I18nProvider from 'core/i18n/Provider';
 import SearchResult from 'amo/components/SearchResult';
+import { getFakeI18nInst } from 'tests/client/helpers';
+
 
 describe('<SearchResult />', () => {
+  function renderResult(result) {
+    return findRenderedComponentWithType(renderIntoDocument(
+      <I18nProvider i18n={getFakeI18nInst()}>
+        <SearchResult result={result} lang="en-US" />
+      </I18nProvider>
+    ), SearchResult).getWrappedInstance();
+  }
+
   const result = {
+    authors: [
+      { name: 'A funky déveloper' },
+      { name: 'A groovy developer' },
+    ],
+    average_daily_users: 553,
     name: 'A search result',
     slug: 'a-search-result',
   };
-  const root = renderIntoDocument(
-    <SearchResult result={result} lang="en-US" />);
+  const root = renderResult(result);
+
+  it('renders the heading', () => {
+    const heading = findRenderedDOMComponentWithClass(root,
+                                                      'SearchResult-heading');
```

Comment:
Nah, you convinced me in that thread. Obviously we'll need to do an audit of the existing code base but I'll change this one as I'm tweaking a few CSS names before the merge anyhow :-)